packages/web/lib/browser.js

```diff
@@ -37,17 +68,15 @@ const getBrowserScope = (parent) => ({
         key,
         functions: {
           page:
-            "Defines a page definition which matches on the regex passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
-          visit: "Open a new tab/page with the given url",
+            "Instantiates a page definition which matches on the regex passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
```

Comment:
it's not a regex, it's a "url-pattern" now. 

I should've updated it in my PR. can you change that now?

Reply:
not resolved